### DISCUSSION TITLE
Error in the β-Strand Homodimer Design tutorial with the homodimer_design command

### DISCUSSION THREAD

**Discussion by danko057** [RosettaCommons/rosetta#112] [General]
[ROSETTA_CRASH.log](https://github.com/user-attachments/files/16563387/ROSETTA_CRASH.log)
On the 3rd step described it in the homodimer design, after running the make_symmdef_file.pl, I obtained those files listed in the tutorial except 2a7b_mpm_A806_anti_wind_1_step_1.kin. But it seem like it doesn't matter since the next command for homodimer_design only require to feed the 2a7b_mpm_A806_anti_wind_1_step_1_INPUT.pdb and symmdef, so I kept going down the step. Apparently, I got the initital error said 

```
ERROR: Assertion `task_is_valid( pose )` failed.
ERROR:: Exit from: src/protocols/minimization_packing/PackRotamersMover.cc line: 300
protocols.jd2.JobDistributor: [ ERROR ] 

[ERROR] Exception caught by JobDistributor for job 2a7b_mpm_A806_anti_wind_1_step_1_INPUT_0001

[ ERROR ]: Caught exception:


File: src/protocols/minimization_packing/PackRotamersMover.cc:300
[ ERROR ] UtilityExitException
ERROR: Assertion `task_is_valid( pose )` failed.
```

And the more following error report which included in the crash log file. Could anybody know how to fix this?

Thanks so much

**Comment by roccomoretti** [comment]
I can confirm this is a bug. You can work around it by changing the options to `-pack_min_runs 1`, but an actual fix that preserves scientific behavior needs a code change.

I've posted a fix for it with PR #115 -- You can apply that patch manually, or you can wait until it's merged into main and then grab and recompile the development version.

**Reply by danko057** [reply]
It appeared that the patch you recommended, the PR #115, would only works if the -pack_min_runs 1, not on any other number. When put any other number, it will come with the same error message.
`ERROR: Assertion `task_is_valid( pose )` failed.
ERROR:: Exit from: src/protocols/minimization_packing/PackRotamersMover.cc line: 300
protocols.jd2.JobDistributor: [ ERROR ] 

[ERROR] Exception caught by JobDistributor for job 2a7b_mpm_A806_anti_wind_1_step_1_INPUT_0007

[ ERROR ]: Caught exception:


File: src/protocols/minimization_packing/PackRotamersMover.cc:300
[ ERROR ] UtilityExitException
ERROR: Assertion `task_is_valid( pose )` failed.

`

**Reply by danko057** [reply]
Adding CRASH.log with the out level 300 just in case if anyone could diagnose the problem.

[ROSETTA_CRASH.log](https://github.com/user-attachments/files/16983360/ROSETTA_CRASH.log)
